packages/httpsnippet-client-api/src/index.js

```diff
@@ -245,7 +245,6 @@ module.exports = function (source, options) {
 
   code
     .push(`sdk${accessor}(${args.join(', ')})`)
-    .push(1, '.then(res => res.json())')
     .push(1, '.then(json => console.log(json))')
```

Comment:
For endpoints that return non-JSON content, what should we do here? Change this part of the snippet to `.push(1, '.then(res => console.log(res))')` and only mention "json" if the SDK is doing this autoretrieval of `res.json()`?

Reply:
Fix we spoke about on Slack https://github.com/readmeio/api/pull/240/commits/4e654ae541269266b6b6a81d34bd9b0e10351bcb